AutomatedLabWorker/AutomatedLabVMWareWorkerNetwork.psm1

```diff
@@ -8,7 +8,7 @@ function Get-LWVMWareNetworkSwitch
 	
     Write-LogFunctionEntry
 
-    foreach ($network in $VirtualNetwork)
+    foreach ($name in $VirtualNetwork)
```

Comment:
The type AutomatedLab.VirtualNetwork has the property Name, which should be used here to improve readability. The overloaded ToString() method returns the network name, but to ensure that a change in the ToString() method does not break stuff, please use the property name. `Get-VDPortgroup -Name $name.Name`